src/styles/blocks/index.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+}
+
+main {
+  --grid-colum: 4;
+  display: grid;
+  grid-template-columns: repeat(var(--grid-colum), 200px);
+  gap: 46px 48px;
+  max-width: 944px;
+  margin: auto;
+  padding-top: 50px;
+  padding-bottom: 72px;
```

Comment:
try to play with padding to pass all tests because I suppose the problem is here.